net/pfSense-pkg-freeradius2/files/usr/local/pkg/googleauth.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/local/bin/python2.7
```

Comment:
A change on Makefile will need to be make to replace shebang line, see more information at https://www.freebsd.org/doc/en/books/porters-handbook/uses-shebangfix.html